cmd/end2endtest/overwritenotwait.go

```diff
@@ -18,7 +16,7 @@ import (
 
 func init() {
 	ops["overwritenotwaitnextblock"] = operation{
-		test:        &E2EAnonElection{},
+		test:        &E2EOverwriteNotWaitElection{},
```

Comment:
i'd suggest naming these tests
"E2EVoteOverwriteInSameBlock" and "E2EVoteOverwriteInNextBlock"
(dropping "Election" as well since the key point here is vote overwrite, the election is kind of a side-effect)

BUT, wouldn't even be simpler to merge both tests (SameBlock and NextBlock) into overwrite.go?
i mean during the E2EOverwriteElection (or E2EVoteOverwriteElection), say, make t.voterAccounts[0] overwrite in the same block and t.voterAccounts[1] in next block.

maybe there's a reason to keep the tests separate, but if not, i believe merging them makes sense